### DISCUSSION TITLE
TorchMetrics - Accuracy

### DISCUSSION THREAD

**Discussion by Bjoern-2** [mrdbourke/pytorch-deep-learning#444] [Q&A] [answered]
Hi, my problem is the  following:
***
from torchmetrics import Accuracy

accuracy = Accuracy().to(device)
accuracy(y_preds, y_blob_test)
***
TypeError: Accuracy.__new__() missing 1 required positional argument: 'task'

I did the exact same thing as in the video (min: 13:57:00) but it doesn't work.

I think I have to put a parameter between the paranthesis like task=task (like task is the amount of classes we have), but it still doesn't work.

Do somebody have an idea?

**Comment by mrdbourke** [comment, marked as answer]
Hi @Bjoern-2,

This was a problem with the version of `TorchMetrics` introducing the `task` parameter.

See the code fix here: https://github.com/mrdbourke/pytorch-deep-learning/discussions/318
